test/e2e/storage/drivers/csi.go

```diff
@@ -236,7 +237,31 @@ func (h *hostpathCSIDriver) PrepareTest(f *framework.Framework) (*storageframewo
 		NodeName:                 node.Name,
 	}
 	cleanup, err := utils.CreateFromManifests(config.Framework, driverNamespace, func(item interface{}) error {
-		return utils.PatchCSIDeployment(config.Framework, o, item)
+		if err := utils.PatchCSIDeployment(config.Framework, o, item); err != nil {
+			return err
+		}
+
+		// Remove csi-external-health-monitor-agent and
+		// csi-external-health-monitor-controller
```

Comment:
@xing-yang can you follow up on scale-testing and re-enabling the health monitor controller sidecar? My understanding is that only the node agent was redesigned due to scalability issues.

Reply:
See https://github.com/kubernetes/kubernetes/pull/102627 - that should go into today's job run.